packages/tupaia-web/src/features/DashboardItem/ExpandItemButton.tsx

```diff
@@ -58,7 +58,12 @@ const ZoomInIcon = styled(MuiZoomIcon)`
   }
 `;
 
-const EXPANDABLE_TYPES = ['chart', 'dataDownload', 'filesDownload'];
+const EXPANDABLE_TYPES = [
+  DashboardItemTypes.Chart,
+  ViewTypes.DataDownload,
+  ViewTypes.FilesDownload,
+  ViewTypes.MultiValue,
+];
```

Comment:
Both @jaskfla and I were having issues with importing this enum here. I've been digging around for a bit trying to work out what's been going wrong. Honestly it has me kinda stumped, I think it's an issue with vite. I might take a bit more of a look, but if we can't solve it perhaps we take another approach?

Reply:
Hmmmm, I'm not getting this. If you restart the TS server in VS Code does it still happen?